pkg/fake/ec2api.go

```diff
@@ -381,21 +374,24 @@ func (e *EC2API) DescribeLaunchTemplatesWithContext(_ context.Context, input *ec
 		return output, nil
 	}
 	if len(output.LaunchTemplates) == 0 {
-		return nil, awserr.New("InvalidLaunchTemplateName.NotFoundException", "not found", nil)
+		return nil, &smithy.GenericAPIError{
+			Code:    "InvalidLaunchTemplateName.NotFoundException",
```

Comment:
checking: This code hasn't changed with the v2 update? I'm assuming not since it's a server-side error code

Reply:
No it did not

Reply:
Yes it is I ran through to make sure